test-framework/common/src/main/java/io/quarkus/test/common/DefaultDockerContainerLauncher.java

```diff
@@ -59,7 +65,7 @@ public void start() throws IOException {
             System.out.println("Pulling container image '" + containerImage + "'");
             try {
                 int pullResult = new ProcessBuilder().redirectError(DISCARD).redirectOutput(DISCARD)
-                        .command(List.of("docker", "pull", containerImage).toArray(new String[0])).start().waitFor();
+                        .command(List.of(DOCKER_BINARY, "pull", containerImage).toArray(new String[0])).start().waitFor();
```

Comment:
Maybe this would look better? 

```suggestion
                        .command(DOCKER_BINARY, "pull", containerImage).start().waitFor();
```

Reply:
Sure, done